src/App.jsx

```diff
@@ -1,104 +1,26 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+function getComentsById(postId) {
+  return commentsFromServer.filter(coment => coment.postId === postId) || null;
```

Comment:
There is a typo in the variable name `coment`. It should be `comment` to maintain consistency with the data structure and improve code readability.